.github/workflows/release.yml

```diff
@@ -109,6 +109,25 @@ jobs:
       - name: Publish to galaxy
         run: ansible-galaxy collection publish --api-key=${{ secrets.GALAXY_INFRA_KEY }} ${{ steps.build.outputs.tar_file }}
 
+      - name: Update dependency for Automation Hub
+        run: 'sed -i ''s/fedora\.linux_system_roles/redhat.linux_system_roles/g'' galaxy.yml'
```

Comment:
Should be `redhat.rhel_system_roles` instead of `redhat.linux_system_roles` everywhere.